IFC4x3/Properties/p/PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx" Name="PermeabilityHoz" UniqueId="264100d7-f3ba-4e86-91ea-e10924edebbb" PrimaryDataType="IfcLinearVelocityMeasure" />
```

Comment:
As with other permeability properties, let us not use this measure. Maybe discuss what to do before using this data type.

Also, propose changing property name to "PermeabilityHorizontal".

Reply:
Regarding naming, I agree! Regarding datatype, discuss with @j-weil.

Reply:
Renamed to HydraulicPermeabilityHorizontal/Vertical. Regarding datatype @j-weil persists in IfcLInearVelocityMeasure. Pset_InSituTestHydro is already merged with a HydraulicPermeability which also had IfcLInearVelocityMeasure.